components/openzwave-adapter/src/lib.rs

```diff
@@ -204,12 +204,13 @@ fn ensure_directory<T: AsRef<Path> + ?Sized>(directory: &T) -> Result<(), Error>
 }
 
 impl OpenzwaveAdapter {
-    pub fn init<T: AdapterManagerHandle + Send + Sync + 'static> (box_manager: &Arc<T>, user_path: &str, device: Option<String>) -> Result<(), Error> {
+    pub fn init<T: AdapterManagerHandle + Send + Sync + 'static> (box_manager: &Arc<T>, user_path: &str, devices: Option<String>) -> Result<(), Error> {
 
         try!(ensure_directory(user_path));
 
         let options = InitOptions {
-            device: device,
+            // We treat devices as a whitespace delimited list of device names.
+            devices: devices.map(|s| s.split_whitespace().map(|s| s.to_owned()).collect()),
```

Comment:
I just wonder why whitespace and not eg comma + space trim ? That seems more natural to me (but maybe that's just me).

also, I wonder if `.cloned()` would work instead of your `.map(...)` ?